src/libs/svg/lv_svg_render.h

```diff
@@ -11,6 +11,12 @@
  *********************/
 #include "../../lv_conf_internal.h"
 
+#if LV_USE_SVG
+    #if !LV_USE_VECTOR_GRAPHIC
+        #error "LV_USE_SVG requires LV_USE_VECTOR_GRAPHIC = 1"
+    #endif
+#endif
+
 #if LV_USE_SVG && LV_USE_VECTOR_GRAPHIC
```

Comment:
You suggestion also works, but we usually use this pattern:
```c
#if LV_USE_SVG

#if !LV_USE_VECTOR_GRAPHIC
#error "LV_USE_SVG requires LV_USE_VECTOR_GRAPHIC = 1"
#endif
```

see [here](https://github.com/lvgl/lvgl/blob/master/src/widgets/slider/lv_slider.h#L21-L23) as a refrerence.

Reply:
oh i like that! pushed, this is now untested